src/components/userSettings/AccessTokenListGroupItem.jsx

```diff
@@ -0,0 +1,82 @@
+/* eslint-env browser */
+import React, { useRef } from 'react'
+import PropTypes from 'prop-types'
+import {
+  ListGroupItem,
+  Alert,
+  InputGroup,
+  Input,
+  InputGroupAddon,
+  Button,
+} from 'reactstrap'
+
+const AccessTokenListGroupItem = props => {
+  const inputRef = useRef()
+
+  const copyValue = e => {
+    inputRef.current.select()
+    document.execCommand('copy')
+    e.target.focus()
+  }
+
+  return (
+    <ListGroupItem>
+      <div className="d-flex flex-row align-items-center">
+        <div className="d-flex flex-column">
+          <strong>{props.name}</strong>
+          <span className="text-muted">Created at {props.createdAt}</span>
+          <span className="text-muted">
+            {props.lastUsedAt
+              ? `Last used at ${props.lastUsedAt}`
```

Comment:
I'm not too opinionated on this, but maybe we could format the date to be a bit nicer?

Reply:
I changed it to show dates the same way we do for questions.